modules/orca/main.tf

```diff
@@ -1,3 +1,8 @@
+locals {
+  # Used for Load Balancer, EC2 Service, and Container ports. Specifies how GQL will be hosted.
+  deploy_graphql = false
```

Comment:
Theoretically could make this a variable at the top level, but didn't feel necessary for a temporary flag; customers shouldn't be seeing this anyway.